tools/Google.Cloud.Tools.ReleaseManager/GenerateApisCommand.cs

```diff
@@ -383,7 +382,7 @@ private int ExecuteForUnconfigured(string[] args)
     private void GenerateUnconfigured(string arg)
     {
         Console.WriteLine($"{DateTime.UtcNow:yyyy-MM-dd'T'HH:mm:ss.fff}Z Generating {arg}");
-        var apiDirectory = Path.Combine(_rootLayout.GeneratorOutput, arg);
```

Comment:
This is deliberately a change btw - it was broken before. (It should have been relative to _rootLayout.Googleapis.)